src/Logger.ts

```
@@ -152,6 +166,23 @@ export default class Logger {
       this.ignoredHosts = new Set(options.ignoredHosts);
     }
 
+    if (options?.ignoredPatterns) {
+      this.ignoredPatterns = options.ignoredPatterns;
+    }
+
+    if (options?.ignoredUrls) {
+      if (
+        !Array.isArray(options.ignoredUrls) ||
+        typeof options.ignoredUrls[0] !== 'string'
+      ) {
+        warn(
```

this may be better as a `throw Error()` rather than a warning.  This will prompt the developer to fix their code.

---

Yeah, I can't remember why I did the warning as it was a while ago when I set it up. I would probably accept a PR updating it.